soe/Gruntconfig.json

```diff
@@ -20,7 +20,7 @@
   },
   "eslint": true,
   "drush": {
-    "path": "/Users/boznik/.composer/vendor/bin/drush",
+    "path": "./vendor/bin/drush",
```

Comment:
If you take only one thing from this PR let it be this line.

Reply:
Ah man. I just renamed my mac user to `boznik`.

Reply:
don't forget to update the README: https://github.com/SU-SOE/grunt-drupal-tasks/blob/SOE-develop-mc/docs/soe_build_notes.md